components/automate-ui/src/app/entities/userperms/userperms.effects.ts

```diff
@@ -60,11 +62,22 @@ export class UserPermEffects {
   fetchSomePerms$ = this.actions$.pipe(
     ofType(UserPermsTypes.GET_SOME),
     withLatestFrom(this.store.select(getlastFetchTime)),
-    filter(([_action, lastTime]) => this.stale(lastTime)),
```

Comment:
Instead of filtering for staleness, do the same check in the body of the effect below, then issue a new action--either one that returns immediately or one that fetches data with a network call.